webpack/helpers/toastHelper.js

```diff
@@ -0,0 +1,3 @@
+import { addToast } from 'foremanReact/redux/actions/toasts';
+
+export const showToast = dispatch => toast => dispatch(addToast(toast));
```

Comment:
why not dispatching `addToast` directly?

Reply:
It would require passing `dispatch` around so I could do `dispatch(addToast({ ... }))`, it just seemed more readable to prepare that in one place and then just call `showToast({ ... })`.